### PR TITLE
Rework of Clothing Item - nt_skirtsuit

### PR DESCRIPTION
**Pull request by deGurkin**
Added 2 new Variants:
a Recolourable unbranded version
a Command version.

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 1]
```diff
@@ -68,6 +68,20 @@
...
+/datum/gear/uniform/suitskirt
+	display_name = "Skirtsuit (recolourable)"
+	description = "A recolourable skirtsuit"
+	path = /obj/item/clothing/under/suit_jacket/skirtsuit/color
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+
+/datum/gear/uniform/command_suitskirt
+	display_name = "Command Skirtsuit"
+	description = "A Skirtsuit for Women in a Command Role"
```
Comment (listerla) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm:80]: Personally, I

1. Wouldn't specify gender
2. Wouldn't capitalize each of these words. Maybe put a period at the end.

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -218,6 +218,23 @@
...
+	desc = "A black coat with an colour coordinated kerchief accompanied by a swept skirt with a tasteful stripe. Works for every occasion."
+	icon_state = "skirtsuit_recolourable"
+	item_state = "skirtsuit_recolourable"
+	worn_state = "skirtsuit_recolourable"
+	build_from_parts = TRUE
+	worn_overlay = "base"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+
+/obj/item/clothing/under/suit_jacket/skirtsuit_command
+	desc = "A black coat with a gold kerchief accompanied by a swept skirt with a respectable golden trim. Works only for Women in Power."
```
Comment (listerla) [inline code/modules/clothing/under/miscellaneous.dm:231]: Ditto with before; I'd either remove or not capitalize 'women' and 'power' shouldn't be capitalized either.

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 3]
```diff
@@ -68,6 +68,20 @@
 	skirts["jumper dress"] = /obj/item/clothing/under/skirt/jumper_highcut
 	gear_tweaks += new /datum/gear_tweak/path(skirts)
 
+/datum/gear/uniform/suitskirt
+	display_name = "Skirtsuit (recolourable)"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm:72]:
```suggestion
	display_name = "skirtsuit (recolourable)"
```

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 4]
```diff
@@ -68,6 +68,20 @@
...
+	description = "A recolourable skirtsuit"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm:73]:
```suggestion
	description = "A recolourable skirtsuit."
```

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 5]
```diff
@@ -68,6 +68,20 @@
...
+	path = /obj/item/clothing/under/suit_jacket/skirtsuit/color
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+
+/datum/gear/uniform/command_suitskirt
+	display_name = "Command Skirtsuit"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm:79]:
```suggestion
	display_name = "command skirtsuit"
```

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm (hunk 1 above)
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm:80]:
```suggestion
	description = "A combined suit with a skirt for people in command."
```

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 6]
```diff
@@ -68,6 +68,20 @@
...
+	description = "A Skirtsuit for Women in a Command Role"
+	path = /obj/item/clothing/under/suit_jacket/skirtsuit_command
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION
+
+
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm:84]: Whitespace.

File: code/modules/clothing/under/miscellaneous.dm [hunk 7]
```diff
@@ -218,6 +218,23 @@
...
+	icon_state = "skirtsuit_command"
+	item_state = "skirtsuit_command"
+	worn_state = "skirtsuit_command"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+
+
```
Comment (SleepyGemmy) [inline code/modules/clothing/under/miscellaneous.dm:237]: Whitespace.

File: code/modules/clothing/under/miscellaneous.dm (hunk 2 above)
Comment (SleepyGemmy) [inline code/modules/clothing/under/miscellaneous.dm:231]:
```suggestion
	desc = "A black coat with a gold kerchief accompanied by a swept skirt with a respectable golden trim. Works for every occasion."
```

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm (hunk 1 above)
Comment (deGurkin) [reply code/modules/client/preference_setup/loadout/loadout_uniform.dm:80]: Good call. 
For a variety of reasons.
Altered in my next update.
grafik

File: code/modules/clothing/under/miscellaneous.dm (hunk 2 above)
Comment (deGurkin) [reply code/modules/clothing/under/miscellaneous.dm:231]: Will be adjusted in next update.
grafik

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm (hunk 3 above)
Comment (deGurkin) [reply code/modules/client/preference_setup/loadout/loadout_uniform.dm:72]: huh, never realised none of the display names are not capitalised.

File: code/modules/clothing/under/miscellaneous.dm (hunk 2 above)
Comment (deGurkin) [reply code/modules/clothing/under/miscellaneous.dm:231]: This was a tongue in Cheek joke intentionally put here to imply that if you are wearing this you better be a Head of Staff or you will have to answer for yourself
I hope this tongue in cheek humor can stay, if not then I will change it.

Comment (SleepyGemmy) [reply code/modules/clothing/under/miscellaneous.dm]: `Works only for people in power."` would work too, of course.

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 8]
```diff
@@ -68,6 +68,18 @@
...
 
+/datum/gear/uniform/suitskirt
+	display_name = "skirtsuit (recolourable)"
+	description = "A recolourable skirtsuit."
+	path = /obj/item/clothing/under/suit_jacket/skirtsuit/color
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/uniform/command_suitskirt
+	display_name = "command skirtsuit"
+	description = "A Skirtsuit for staff in a command role."
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm:79]:
```suggestion
	description = "A skirtsuit for staff in a command role."
```

Comment (deGurkin) [reply code/modules/client/preference_setup/loadout/loadout_uniform.dm:79]: The rules of the english language elude me I am terribly sorry.
grafik

**Review by SleepyGemmy — APPROVED** [review]
Changes addressed. Language and grammar looks fine. Code looks fine, to my knowledge.
_(Note that this doesn't mean the PR is approved, since I'm not a maintainer.)_

**Comment by deGurkin** [comment]
!review

**Comment by deGurkin** [comment]
panic

**Comment by NonQueueingMatt** [comment]
Inactive, DM me if you want this reopened.

**Pull request closed without merging**